Sources/ShortcutRecorder/SRShortcut.m

```diff
@@ -203,6 +203,15 @@ - (instancetype)initWithCode:(SRKeyCode)aKeyCode
     return self;
 }
 
+#pragma mark Non-initializer class methods
+
++ (BOOL)sequoiaValidModifiers:(NSEventModifierFlags)modifierFlags
```

Comment:
I wouldn't name it with 'Sequoia' in it, as it'll be out of date next year.
I wonder if we bring the `if @available()` check in here for simplicity? So we have a single entry point that is `validModifiers:`?